shared/actions/wallets.js

```diff
@@ -29,12 +31,20 @@ const buildPayment = (state: TypedState, action: any) =>
     toIsAccountID:
       state.wallets.buildingPayment.recipientType !== 'keybaseUser' &&
       !Constants.isFederatedAddress(state.wallets.buildingPayment.to),
-  }).then(build =>
-    WalletsGen.createBuiltPaymentReceived({
-      build: Constants.buildPaymentResultToBuiltPayment(build),
-      forBuildingPayment: state.wallets.buildingPayment,
+  })
+    .then(build =>
+      WalletsGen.createBuiltPaymentReceived({
+        build: Constants.buildPaymentResultToBuiltPayment(build),
+        forBuildingPayment: state.wallets.buildingPayment,
+      })
+    )
+    .catch(error => {
+      if (error instanceof RPCError && error.code === RPCTypesKb.constantsStatusCode.sccanceled) {
+        // ignore cancellation
+      } else {
+        throw error
+      }
```

Comment:
I get this (with a logger.info in place of the "ignore" comment above). Want to confirm that those values aren't used.

![image](https://user-images.githubusercontent.com/705646/46438798-d1f84f80-c72c-11e8-8a9f-e1b2788d065c.png)